tutorials/kubernetes-demo/blockgen/main.go

```diff
@@ -300,3 +359,81 @@ type gen interface {
 	Ts() int64
 	Value() float64
 }
+
+//
+//func genTSDBBlock(
```

Comment:
if you intend to leave this here I'd suggest we put at least a comment there explaining why it's here and why it's commented out

Reply:
I am experimenting a lot on this branch, so will close this PR for now to avoid confusion